twitter copy.css

```diff
@@ -0,0 +1,251 @@
+.left {
```

Comment:
You can add the font you want to use for the whole page on the body tag, then change it accordingly.